fdroid/config.yml

```diff
@@ -0,0 +1,18 @@
+---
+repo_url: https://mobileapp.bitwarden.com/fdroid/repo/
```

Comment:
Should we set these URLs to `http` to F-Droid stops complaining when we refresh apps?

Reply:
Disregard that. I misread the Toast from F-Droid. Found the actual issue in [archive/index.html](https://github.com/bitwarden/f-droid/blob/main/fdroid/archive/index.html#L41). Hopefully it is a typo in the old config causing this. 🤞